relay-metrics/src/meta/protocol.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
+
+use chrono::{DateTime, Utc};
+use relay_common::time::UnixTimestamp;
+use serde::{Deserialize, Serialize};
+
+use crate::MetricResourceIdentifier;
+
+/// A metric metadata item.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct MetricMeta {
+    /// Timestamp scope for the contained metadata.
+    ///
+    /// Metric metadata is collected in daily intervals, so this may be truncated
+    /// to the start of the day (UTC) already.
+    pub timestamp: StartOfDayUnixTimestamp,
```

Comment:
@jan-auer do we have to do any validation to make sure this is *not* in the future?